<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/image/5624330_201027415000_2.jpg</normaloff>:/image/5624330_201027415000_2.jpg</iconset>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDraw">
    <property name="title">
     <string>Draw</string>
    </property>
    <addaction name="actionCurve"/>
    <addaction name="actionCloseCurve"/>
    <addaction name="actionPolyline"/>
    <addaction name="actionPolygon"/>
    <addaction name="actionCircle"/>
    <addaction name="actionRect"/>
    <addaction name="actionEllipse"/>
    <addaction name="actionText"/>
   </widget>
   <widget class="QMenu" name="menuPick">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionPick"/>
    <addaction name="actionPalm"/>
   </widget>
   <widget class="QMenu" name="menuFix">
    <property name="title">
     <string>Fit</string>
    </property>
    <addaction name="actionFixcanvas"/>
    <addaction name="actionExpand"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionAdd"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
   </widget>
   <widget class="QMenu" name="menuCombination">
    <property name="title">
     <string>Combination</string>
    </property>
    <addaction name="actionCombination"/>
    <addaction name="actionDivide"/>
    <addaction name="actionDivideToEnd"/>
   </widget>
   <widget class="QMenu" name="menuZoom">
    <property name="title">
     <string>Zoom</string>
    </property>
    <addaction name="actionZoomIn"/>
    <addaction name="actionZoomOut"/>
    <addaction name="actionZoomOne"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionMoveToTop"/>
    <addaction name="actionMoveToBottom"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
   </widget>
   <widget class="QMenu" name="menuAaa">
    <property name="title">
     <string>aaa</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuDraw"/>
   <addaction name="menuPick"/>
   <addaction name="menuFix"/>
   <addaction name="menuCombination"/>
   <addaction name="menuZoom"/>
   <addaction name="menuAaa"/>
  </widget>
  <widget class="QToolBar" name="toolBar_0">
   <property name="windowTitle">
    <string>File</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionAdd"/>
   <addaction name="actionSave"/>
   <addaction name="actionSaveAs"/>
  </widget>
  <widget class="QToolBar" name="toolBar_1">
   <property name="windowTitle">
    <string>Draw</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionCurve"/>
   <addaction name="actionCloseCurve"/>
   <addaction name="actionPolyline"/>
   <addaction name="actionPolygon"/>
   <addaction name="actionCircle"/>
   <addaction name="actionRect"/>
   <addaction name="actionEllipse"/>
   <addaction name="actionText"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>Edit</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionPick"/>
   <addaction name="actionCut"/>
   <addaction name="actionCopy"/>
   <addaction name="actionPaste"/>
   <addaction name="actionMoveToTop"/>
   <addaction name="actionMoveToBottom"/>
   <addaction name="actionFixcanvas"/>
   <addaction name="actionExpand"/>
   <addaction name="actionZoomIn"/>
   <addaction name="actionZoomOut"/>
   <addaction name="actionZoomOne"/>
   <addaction name="actionCombination"/>
   <addaction name="actionDivide"/>
   <addaction name="actionDivideToEnd"/>
   <addaction name="actionPalm"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionCurve">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/image/pencapstyle.png</normaloff>:/image/pencapstyle.png</iconset>
   </property>
   <property name="text">
    <string>Curve</string>
   </property>
  </action>
  <action name="actionPolyline">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/image/polyline.png</normaloff>:/image/polyline.png</iconset>
   </property>
   <property name="text">
    <string>Polyline</string>
   </property>
  </action>
  <action name="actionPolygon">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/image/polygon.png</normaloff>:/image/polygon.png</iconset>
   </property>
   <property name="text">
    <string>Polygon</string>
   </property>
  </action>
  <action name="actionPick">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/image/th.jpg</normaloff>:/image/th.jpg</iconset>
   </property>
   <property name="text">
    <string>Pick</string>
   </property>
  </action>
  <action name="actionCircle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/image/circle.PNG</normaloff>:/image/circle.PNG</iconset>
   </property>
   <property name="text">
    <string>Circle</string>
   </property>
  </action>
  <action name="actionFixcanvas">
   <property name="icon">
    <iconset>
     <normaloff>:/image/fit.jpg</normaloff>:/image/fit.jpg</iconset>
   </property>
   <property name="text">
    <string>Fitcanvas</string>
   </property>
  </action>
  <action name="actionRect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/image/rect.png</normaloff>:/image/rect.png</iconset>
   </property>
   <property name="text">
    <string>Rect</string>
   </property>
  </action>
  <action name="actionEllipse">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/image/ellipse.png</normaloff>:/image/ellipse.png</iconset>
   </property>
   <property name="text">
    <string>Ellipse</string>
   </property>
  </action>
  <action name="actionText">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/image/text.png</normaloff>:/image/text.png</iconset>
   </property>
   <property name="text">
    <string>Text</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset>
     <normaloff>:/image/open.png</normaloff>:/image/open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset>
     <normaloff>:/image/save.png</normaloff>:/image/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionAdd">
   <property name="icon">
    <iconset>
     <normaloff>:/image/blue-plus-sign-md.png</normaloff>:/image/blue-plus-sign-md.png</iconset>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </action>
  <action name="actionCombination">
   <property name="icon">
    <iconset>
     <normaloff>:/image/combine.jpg</normaloff>:/image/combine.jpg</iconset>
   </property>
   <property name="text">
    <string>Combination</string>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="icon">
    <iconset>
     <normaloff>:/image/zoomin.png</normaloff>:/image/zoomin.png</iconset>
   </property>
   <property name="text">
    <string>ZoomIn</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="icon">
    <iconset>
     <normaloff>:/image/zoomout.png</normaloff>:/image/zoomout.png</iconset>
   </property>
   <property name="text">
    <string>ZoomOut</string>
   </property>
  </action>
  <action name="actionPalm">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/image/hand.jpg</normaloff>:/image/hand.jpg</iconset>
   </property>
   <property name="text">
    <string>Palm</string>
   </property>
  </action>
  <action name="actionZoomOne">
   <property name="icon">
    <iconset>
     <normaloff>:/image/zoomone.png</normaloff>:/image/zoomone.png</iconset>
   </property>
   <property name="text">
    <string>ZoomOne</string>
   </property>
  </action>
  <action name="actionMoveToTop">
   <property name="icon">
    <iconset>
     <normaloff>:/image/up.png</normaloff>:/image/up.png</iconset>
   </property>
   <property name="text">
    <string>MoveToTop</string>
   </property>
  </action>
  <action name="actionMoveToBottom">
   <property name="icon">
    <iconset>
     <normaloff>:/image/down.png</normaloff>:/image/down.png</iconset>
   </property>
   <property name="text">
    <string>MoveToBottom</string>
   </property>
  </action>
  <action name="actionCloseCurve">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/image/5624330_201027415000_2.jpg</normaloff>:/image/5624330_201027415000_2.jpg</iconset>
   </property>
   <property name="text">
    <string>CloseCurve</string>
   </property>
  </action>
  <action name="actionDivide">
   <property name="icon">
    <iconset>
     <normaloff>:/image/divide.jpg</normaloff>:/image/divide.jpg</iconset>
   </property>
   <property name="text">
    <string>Divide</string>
   </property>
  </action>
  <action name="actionDivideToEnd">
   <property name="icon">
    <iconset>
     <normaloff>:/image/dividetoend.jpg</normaloff>:/image/dividetoend.jpg</iconset>
   </property>
   <property name="text">
    <string>DivideToEnd</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset>
     <normaloff>:/image/new.png</normaloff>:/image/new.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset>
     <normaloff>:/image/copy.png</normaloff>:/image/copy.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset>
     <normaloff>:/image/paste.png</normaloff>:/image/paste.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionExpand">
   <property name="icon">
    <iconset>
     <normaloff>:/image/expand.jpg</normaloff>:/image/expand.jpg</iconset>
   </property>
   <property name="text">
    <string>Expand</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset>
     <normaloff>:/image/cut.png</normaloff>:/image/cut.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset>
     <normaloff>:/image/undo.png</normaloff>:/image/undo.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset>
     <normaloff>:/image/redo.png</normaloff>:/image/redo.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="icon">
    <iconset>
     <normaloff>:/image/save_all_download-512.png</normaloff>:/image/save_all_download-512.png</iconset>
   </property>
   <property name="text">
    <string>SaveAs</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
